src/components/Renderer/index.tsx

```diff
@@ -18,7 +18,7 @@ import {
 	WidgetMeta,
 } from './widgets';
 import { transformUiSchema, getBestSchemaMatch } from './widgets/widget-util';
-import { Value, JSONSchema, UiSchema, Format, JsonTypes } from './types';
+import type { Value, JSONSchema, UiSchema, Format, JsonTypes } from './types';
```

Comment:
apparently, JsonTypes is an object and we use it as a value not as a type

Reply:
Nice spot :+1: